Runtime/Scripts/Timeline/GLTFRecorder.cs

```diff
@@ -175,7 +175,48 @@ internal void endRecording(out Dictionary<Transform, AnimationData>? param)
 			param = recordingAnimatedTransforms;
 		}
 
-		public void EndRecording(string filename, string sceneName = "scene", GLTFSettings? settings = null)
+		public bool TryEndRecordingDirect()
+		{
+			if (!isRecording) return false;
+			if (!hasRecording) return false;
+			isRecording = false;
+			
+			#if UNITY_EDITOR
+			Debug.Log("Gltf Recording saved. "
+				+ "Tracks: " + recordingAnimatedTransforms.Count + ", "
+				+ "Total Keyframes: " + recordingAnimatedTransforms.Sum(x => x.Value.tracks.Sum(y => y.Values.Count())));
+			#endif
+
+			return true;
+		}
+		
+		public GLTFSceneExporter CreateAnimationExporterAfterRecording(GLTFSettings? settings = null) 
```

Comment:
Since the returned type is called "SceneExporter", i would prefer if the method name used the same naming, i.e. `CreateSceneExporterAfterRecording`.